doc: add v2 backing image doc (#1031)
longhorn/website: content/docs/1.8.0/advanced-resources/system-backup-restore/restore-longhorn-system.md

- Longhorn automatically restores a `Volume` from its latest backup.
- To prevent overwriting eligible settings, Longhorn does not restore the `ConfigMap/longhorn-default-setting`.
- Longhorn does not restore [configurable settings](#configurable-settings).

> - Since Longhorn does not back up the v2 data engine backing image, users should ensure the v2 backing image is prepared in the cluster before restoring the Longhorn system to prevent restoration failures of v2 volumes with the v2 data engine backing images.

Reviewer: ```suggestion
- Since Longhorn does not back up V2 Data Engine backing images, you must ensure that those images are available in the cluster before you restore the Longhorn system. This allows Longhorn to restore volumes that use V2 Data Engine backing images.
```

Author: fixed.
